our-work/workshops-and-talks/jsPsych.yml

```diff
@@ -0,0 +1,16 @@
+title: Converting a jsPsych task with Honeycomb
+description: Guide on creating and converting jsPsych tasks with Honeycomb on Firebase. Workshop was held on August 19, 2022.
+slug: honeycomb
+languages:
+  - reactjs
+  - javascript
+tags:
+  - honeycomb
+people:
+  - name: Ben Heasly
+    github_user: benjamin-heasly
+groups:
```

Comment:
This group doesn't make sense to me.... If it can be empty, lets leave it empty. I'd like to remove that, but I'll need to consult more